### PR TITLE
cool#9992 doc electronic sign: serialize the fetched signature

### PR DESCRIPTION
**Pull request by vmiklos**
- **cool#9992 doc electronic sign: serialize the fetched signature**
- **cool#9992 doc electronic sign: add cypress test**
- **ctags: handle typescript sources**

### CODE REVIEW

File: cypress_test/integration_tests/desktop/draw/esign_spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+			.then(function(win) {
+				cy.stub(win, 'open').as('windowOpen');
+			});
+		cy.intercept('POST', 'https://test.eideasy.com/api/signatures/download-signed-file',
+			{fixture : 'fixtures/eideasy-get-signature.json'}).as('getSignature');
+
+		// When signing that document:
+		cy.cGet('#menu-insert').click();
+		cy.cGet('#menu-insert-esignature').click();
+		cy.wait(['@sendHash'])
```
[the remark this answers is not in the document]
Comment (vmiklos) [reply cypress_test/integration_tests/desktop/draw/esign_spec.js:23]: Let me fix.

**Comment by vmiklos** [comment]
https://cpci.cbg.collabora.co.uk:8080/job/github_online_master_debug_vs_co-24.04/3507/console failed in:

> unit-storage

but these are just JS changes, so that looks unrelated, retry.

**Comment by vmiklos** [comment]
https://cpci.cbg.collabora.co.uk:8080/job/github_online_master_debug_vs_co-24.04_cypress_desktop/2944/console failed in:

> calc/searchbar_spec.js
> calc/top_toolbar_spec.js

retry to see if this is persistent.

**Comment by vmiklos** [comment]
@caolanm could you please review this? Thanks.

A first working scenario with a single provider in the test env, finally including a test, trying to mock actual network traffic.

**Review by caolanm — APPROVED** [review]
That's a quite nice capability of cypress.

**Pull request merged**